### PR TITLE
fvpn: initial stable release v0.2.1

### PR DESCRIPTION
**Pull request by anatolio-deb**
Signed-off-by: Anatolio Nikiforidis anatoly@forestvpn.com

Maintainer: me
Compile tested: main/arm64
Run tested: main/arm64

### CODE REVIEW

File: net/fvpn/Makefile
Comment (BKPepe) [inline net/fvpn/Makefile:48]: Why are you calling the same?

File: net/fvpn/Makefile
Comment (BKPepe) [inline net/fvpn/Makefile:30]: No need to use template.

**Review by BKPepe — CHANGES_REQUESTED** [review]
I briefly checked it and found something what could be improved. Anyway, I am not sure if this can be merged when looking into https://forestvpn.com/pricing/ , so I will leave it to the other guys to decide.

File: net/fvpn/Makefile
Comment (BKPepe) [inline net/fvpn/Makefile:11]: Oh, this is wrong. The tarball will be downloaded as "v$(PKG_VERSION).tar.gz" in dl folder w/o any name.

**Comment by anatolio-deb** [comment]
> I briefly checked it and found something what could be improved. Anyway, I am not sure if this can be merged when looking into https://forestvpn.com/pricing/ , so I will leave it to the other guys to decide.

Hi, thank you for your review. We are going to change our business plan and make some of our resources available fo free and without ads.

File: net/fvpn/Makefile
Comment (1715173329) [inline net/fvpn/Makefile:12]:
```suggestion
PKG_SOURCE_URL:=https://codeload.github.com/forestvpn/cli/tar.gz/v$(PKG_VERSION)?
```

File: net/fvpn/Makefile
Comment (1715173329) [inline net/fvpn/Makefile:30]:
```suggestion

```

File: net/fvpn/Makefile
Comment (1715173329) [inline net/fvpn/Makefile:40]: This description is really tiny, just move it to TITLE.

File: net/fvpn/test.sh
Comment (1715173329) [inline net/fvpn/test.sh:5]: As there only exists one package, the usage of `case` can be removed.

File: net/fvpn/Makefile
Comment (1715173329) [reply net/fvpn/Makefile]: `?` is necessary here

**Comment by 1715173329** [comment]
Final step is to squash all of your commits into a single one.
But looks like repo managers here do not like this package...

**Comment by anatolio-deb** [comment]
> Final step is to squash all of your commits into a single one.
> But looks like repo managers here do not like this package...

Thank you for your feedback. We are going to provide freeware services in this package.

**Pull request closed without merging**
